support/ext/mongo

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-mongo_version="1.6.3"
+mongo_version="1.1.7"
```

Comment:
Actually the latest version of the Mongo PHP driver is 1.1.7 (https://github.com/mongodb/mongo-php-driver/releases/latest). Hence, the previous script was probably not working